data/json/items/ammo.json

```diff
@@ -1440,7 +1440,7 @@
         "name_plural" : "5.56 NATO SS109",
         "symbol" : "=",
         "color" : "yellow",
-        "description" : "5.56x45mm ammunition with a 62gr FMJ bullet containing a steel penetrator. It's light weight, high accuracy and managable recoil lead to it's use by NATO countries and many more. The round's yaw and fragmentation cause significant damage inside a target.",
```

Comment:
Yeah that phrasing is rather clunky. The entire thing could use a rewording.

Also, your English is better than that of a lot of folks who are native speakers here in the USA, so don't sweat it.